### ISSUE TITLE
[ADE Support]: Core Veteran Experience Crew: 508 Audit Report for 44897 Veteran Status Card Findings - Defect 4 | Critical

### ISSUE THREAD

**Issue by lakper** [department-of-veterans-affairs/va.gov-team#117800]
> [!NOTE]
> **QUICK SUMMARY:** _Parent ticket of #117682 ._

### Defect 4 | Critical | Created:
_WCAG 4.1.2 Name, Role, Value | For all user interface components (including but not limited to: form elements, links and components generated by scripts), the name and role can be programmatically determined; states, properties, and values that can be set by the user can be programmatically set; and notification of changes to these items is available to user agents, including assistive technologies._

Throughout the product, the name of the button cannot be understood in relation to the state. For example, on the Veteran Status Card | Veterans Affairs screen (View your Veteran Status Card), under Frequently asked questions, “Expand all accordions Button collapsed” is read. One possibility is to change the name, “Expand all accordions” or “Collapse all accordions”, without indicating the state. Another possibility is to keep the name the same, “all accordions Button collapsed” or “all accordions Button expanded”, and change the state—this is the convention used throughout the product. Note that when using just the arrow keys, JAWS does not read the correct name. After navigating to Expand all accordions and activating it, navigating down followed by up and activating the button, “Expand all accordions” is shown but “Collapse all accordions Button collapsed” is read when reading the current line. A similar example can be found on the Types Of Veteran ID Cards | Veterans Affairs screen (View your Veteran Status Card -> What other types of Veteran ID are available? -> Learn about other types of Veteran ID cards) under Department of Defense Identification Card. (New: 08/12/2025)

## Supporting Artifacts 
- [Project Site](https://staging.va.gov/profile)

### Ticket Updates
- [x] _Connect to an `Epic/Parent` or `Intake` (what body of work is this a part of?) If this is the `Epic/Parent`, connect related tickets as sub-issues._
- [x] _If this is the `Epic/Parent` ticket, add label for `ADE 508 RECHECK`_
- [x] _If this ticket is a `Child` ticket aka `sub-issue` please add the appropriate labels for `ADE 508 FLAG CRITICAL`;`ADE 508 FLAG HIGH`; `ADE 508 FLAG MEDIUM` or `ADE 508 FLAG LOW`_
- [x] _Label with `Originator/Team` (product team or stakeholder requesting support; ADE if internal)_
- [x] _Label date in the `Open Date` field_
- [ ] _Label with `Estimate` (level of effort expected for this ticket)_
- [ ] _Add `Assignee(s)` name(s) to ticket_ (include Lakwi)
- [ ] _Add `Assignee(s) name(s) to each task` they will complete via handle tag (if known)_
- [ ] _Select a `Priority Level`_
- [ ] _Update date in `Last Checked` field_
- [ ] _Label with `Actual` (level of effort it took to complete this ticket)_
- [ ] _Update date in_ `Closed Date`
- [ ] _Make sure the `ADE 508 RECHECK` label is on this ticket upon closing and that a follow-up date has been established and calendared._

**Comment by coforma-jamie** [comment]
Feedback in parent, closing
